packages/notification-services-controller/CHANGELOG.md

```diff
@@ -7,6 +7,14 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.1.3]
+
+### Changed
+
+- Upgrade TypeScript to v5.0 and set `moduleResolution` option to `Node16` ([#3645](https://github.com/MetaMask/core/pull/3645))
+- Bump `@metamask/base-controller` from `^6.0.1` to `^6.0.2` ([#4544](https://github.com/MetaMask/core/pull/4544))
+- Bump `@metamask/controller-utils` from `^11.0.1` to `^11.0.2` ([#4544](https://github.com/MetaMask/core/pull/4544))
```

Comment:
Do we want to include this too?

``` suggestion
- Bump `@metamask/controller-utils` from `^11.0.1` to `^11.0.2` ([#4544](https://github.com/MetaMask/core/pull/4544))
- Bump `@metamask/keyring-controller` from `^17.1.1` to `^17.1.2` ([#4548](https://github.com/MetaMask/core/pull/4548))
- Bump `@metamask/profile-sync-controller` from `^0.1.4` to `^0.1.5` ([#4548](https://github.com/MetaMask/core/pull/4548))

Reply:
- Superseded by https://github.com/MetaMask/core/pull/4556